src/Instrumentation/Psr15/src/Psr15Instrumentation.php

```diff
@@ -24,6 +24,8 @@
  */
 class Psr15Instrumentation
 {
+    public static ContextKey $rootSpanExists;
```

Comment:
I think this should just be called `$rootSpan`

Reply:
ok, It was `$rootSpan` at the beginning, but I changed that due to fact that I was storing bool